commands/commands.go

```diff
@@ -62,6 +62,7 @@ func (c *Command) Run(pctx context.Context, bus *events.EventBus) {
 	c.lock.Lock()
 	log.Debugf("%s.Run start", c.Name)
 	c.setUpCmd()
+	defer reapChildren(c.Cmd.SysProcAttr.Pgid)
```

Comment:
I think the defer helps to make `reapChildren` place within `Command.Run` easier to grok.